### DISCUSSION TITLE
GTK apps slow to start

### DISCUSSION THREAD

**Discussion by KiaraGrouwstra** [YaLTeR/niri#166] [Q&A] [answered]
as mentioned [by someone else as well](https://github.com/YaLTeR/niri/discussions/61#discussioncomment-7417681), GTK apps seem somewhat slow to start on niri.
i'm not sure if this is something about the way i set things up, but i wonder if this is a known issue?
might this pertain to e.g. `xdg-desktop-portal-gtk` / `xdg-desktop-portal-gnome`?

**Comment by YaLTeR** [comment, marked as answer]
That sounds like an issue with the portal setup (it's probably the 10 s DBus timeout). Make sure you're running niri as a session and the portals start up successfully I guess. Make sure you *don't* have `GTK_USE_PORTAL=1` or `GDK_DEBUG=portals` set.
